canonical/store-components
# Wd 2842 refactor package card component

## Done
- Refactored the `PackageCard` component so features can be toggled with props
- Removed `CardList` and `MainPage` components as they are not needed

## QA
- Go to https://store-components-35.demos.haus/
- Check that there are cards for Bundles, Charms, Default and Interfaces

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-2842
